Tests/UnitTests/Identity/CustomerInfoManagerTests.swift

```diff
@@ -217,61 +224,58 @@ class CustomerInfoManagerTests: TestCase {
         let oldInfo = try CustomerInfo(data: [
             "request_date": "2019-08-16T10:30:42Z",
             "subscriber": [
-                "original_app_user_id": "app_user_id",
+                "original_app_user_id": Self.appUserID,
                 "first_seen": "2019-06-17T16:05:33Z",
                 "subscriptions": [:],
                 "other_purchases": [:]
             ]])
 
         var object = try oldInfo.asData()
 
-        let appUserID = "myUser"
-        mockDeviceCache.cachedCustomerInfo[appUserID] = object
+        mockDeviceCache.cachedCustomerInfo[Self.appUserID] = object
 
-        customerInfoManager.sendCachedCustomerInfoIfAvailable(appUserID: appUserID)
+        customerInfoManager.sendCachedCustomerInfoIfAvailable(appUserID: Self.appUserID)
 
         let newInfo = try CustomerInfo(data: [
             "request_date": "2019-08-16T10:30:42Z",
             "subscriber": [
-                "original_app_user_id": "app_user_id",
+                "original_app_user_id": Self.appUserID,
                 "first_seen": "2019-06-17T16:05:33Z",
                 "subscriptions": ["product_a": ["expires_date": "2018-05-27T06:24:50Z", "period_type": "normal"]],
                 "other_purchases": [:]
             ]])
 
         object = try newInfo.asData()
-        mockDeviceCache.cachedCustomerInfo[appUserID] = object
+        mockDeviceCache.cachedCustomerInfo[Self.appUserID] = object
 
-        customerInfoManager.sendCachedCustomerInfoIfAvailable(appUserID: appUserID)
+        customerInfoManager.sendCachedCustomerInfoIfAvailable(appUserID: Self.appUserID)
         expect(self.customerInfoManagerChangesCallCount) == 2
     }
 
     func testSendCachedCustomerInfoIfAvailableForAppUserIDSendsOnMainThread() throws {
         let oldInfo = try CustomerInfo(data: [
             "request_date": "2019-08-16T10:30:42Z",
             "subscriber": [
-                "original_app_user_id": "app_user_id",
+                "original_app_user_id": Self.appUserID,
                 "first_seen": "2019-06-17T16:05:33Z",
                 "subscriptions": [:],
                 "other_purchases": [:]
             ]])
 
         let object = try oldInfo.asData()
-        let appUserID = "myUser"
-        mockDeviceCache.cachedCustomerInfo[appUserID] = object
+        mockDeviceCache.cachedCustomerInfo[Self.appUserID] = object
 
-        customerInfoManager.sendCachedCustomerInfoIfAvailable(appUserID: appUserID)
+        customerInfoManager.sendCachedCustomerInfoIfAvailable(appUserID: Self.appUserID)
         expect(self.mockOperationDispatcher.invokedDispatchOnMainThreadCount) == 1
     }
 
     func testCustomerInfoReturnsFromCacheIfAvailable() {
```

Comment:
All the existing tests are left unchanged, meaning the existing default behavior is still the same.